draft-ietf-core-pubsub.md

```diff
@@ -195,19 +197,32 @@ object in a publish-subscribe system. Topics are conventionally formed
 as a hierarchy, e.g. "/sensors/weather/barometer/pressure" or
 "/EP-33543/sen/3303/0/5700".  The topics are hosted by a Broker and
 all the clients using the Broker share the same namespace for
-topics. Every CoAP pub/sub topic has an associated link, consisting of a reference
+topics. 
+
+Every CoAP pub/sub topic has an associated link, consisting of a reference
 path on the Broker using URI path {{RFC3986}} construction and link
 attributes {{RFC6690}}. Every topic is associated with zero or more
 stored representations and a content-format specified in the link. A
 CoAP pub/sub topic value may alternatively consist of a collection of one or
 more sub-topics, consisting of links to the sub-topic URIs and
 indicated by a link-format content-format. Sub-topics are also topics and
 may have their own sub-topics, forming a tree structure of unique paths that
-is implemented using URIs. The full URI of a topic includes a scheme and authority
-for the Broker, for example "coaps://10.0.0.13:5684/EP-33543/sen/3303/0/5700".
-
-## brokerless Pub/sub
-
+is implemented using URIs. The full URI of a topic includes a scheme and 
+authority for the Broker, for example 
+"coaps://10.0.0.13:5684/EP-33543/sen/3303/0/5700".
+
+A Topic may have a lifetime defined by using the CoAP Max-Age option on topic 
+creation, or on publish operations to the topic. The lifetime is refreshed each 
+time a representation is published to the topic. If the lifetime expires, the 
+topic is removed from discovery responses, returns errors on subscription, and 
+any outstanding subscriptions are cancelled.
```

Comment:
s/outstanding subscriptions are cancelled/outstanding subscriptions for the topic are cancelled by sending a "4.04" response code to subscribers of the topic/

Reply:
This part needs WG discussion